### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by leyen-me** [1Panel-dev/1Panel#5841] [Q&A]
### 联系方式

672228275@qq.com

### 1Panel 版本

进不去

### 问题描述

刚安装上，进不去系统。
![image](https://github.com/user-attachments/assets/eeb82ed2-5e81-4ba8-bcd4-157caf07c4fc)
![image](https://github.com/user-attachments/assets/4aacb5db-11f5-40f0-bdf4-437ae2e2b3ff)

### 重现步骤

http://$LOCAL_IP:37067/login
一直重定向，连登录界面都没有。

**Comment by wanghe-fit2cloud** [comment]
1、取消安全入口，是否能正常登录
2、切换 IPv4 地址，是否能正常登录

1pctl 命令行工具教程：https://1panel.cn/docs/installation/cli/
